docs/reports/migrate_omim.md

```diff
@@ -2,21 +2,6 @@
 [Interactive FlatGithub table](https://flatgithub.com/monarch-initiative/mondo-ingest?filename=src/ontology/slurp/omim.tsv)
 
 ### Migratable terms
-| mondo_id      | mondo_label                                | xref                 | xref_source                | original_label                             | definition    | parents                     |
```

Comment:
The OMIM and ORDO slurp/migrate tables, as well as unmapped OMIM table, are now empty, but I think this is expected based on recent curation, yes?

Reply:
I think so, yes